excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/SpreadsheetValidationErrorMessageCreator.java

```diff
@@ -449,6 +449,24 @@ public String createErrorMessage(EventCaseFieldMetadataValidatorImpl.ValidationE
         return withWorkSheetName(error);
     }
 
+    @Override
+    public String createErrorMessage(UserProfileValidatorImpl.ValidationError validationError) {
+        return withWorkSheetName(validationError);
+    }
+
+    @Override
+    public String createErrorMessage(CaseTypeEntityReferenceSpellingValidationError error) {
+        return newMessageIfDefinitionExists(error,
+            error.getCaseTypeEntity(),
+            def -> String.format(
+                "Case Type with ID '%s' on tab '%s' already exists with the definitive spelling '%s'. "
+                    + "This spelling must be used for the Case Type ID.",
+                error.getCaseTypeEntity().getReference(),
```

Comment:
A little awkward to see String.format and a plus sign in the same statement

Reply:
If we are not careful, we can end up with line like https://github.com/hmcts/ccd-definition-store-api/blob/16c0f885056b45141bf4f64164a3c57ffe1460a3/excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/endpoint/exception/SpreadsheetValidationErrorMessageCreator.java#L116


A suggestion is 

String.format(
                "Case Type with ID '%s' on tab '%s' already exists with the definitive spelling '%s'. %s",
                error.getCaseTypeEntity().getReference(),
                def.getSheetName(),
                error.getDefinitiveCaseTypeReference()),
                "This spelling must be used for the Case Type ID."